frontend/src/components/FileUploadForm.vue

```diff
@@ -69,7 +70,7 @@
                 // }
             },
             encyptedFile(index, contentIndex){
-                if (index===0 && contentIndex === 0){
+                if (index===0 && contentIndex === 0 && !this.uploading){
```

Comment:
Does the index and contentIndex 0 mean anything in particular? I normally ask this when I see numbers in code.

Reply:
It's checking if the first chunk in particular is encrypted. It uploads the file in parallel in chunks, so as soon as it has finished the first chunk it can start uploading while the second chunk is still encrypting, the add for this PR checks that it isn't already uploading.